pages/linux/ruget.md

```diff
@@ -0,0 +1,12 @@
+# ruget
+
+> Alternative to wget written in Rust.
+> More information: <https://github.com/ksk001100/ruget>.
+
+- Download the contents of a URL to a file (named "foo" in this case):
+
+`ruget https://example.com/foo`
+
+- Download the contents of a URL to a file with a specified [o]utput file (named "bar" in this case):
```

Comment:
```suggestion
- Download the contents of a URL to a specified [o]utput file:
```

Reply:
I think the 'specified' is a more precise word in this case. The other suggestions are good!